lib/client.js

```diff
@@ -41,6 +41,7 @@ var Client = function Client(dsn, options) {
   this.root = options.root || process.cwd();
   this.transport = options.transport || transports[this.dsn.protocol];
   this.release = options.release || process.env.SENTRY_RELEASE || '';
+  this.environment = options.environment || process.env.SENTRY_ENVIRONMENT || '';
```

Comment:
> Remove NODE_ENV entirely, fixes many issues since people have different opinions on wtf this means

lol, yeah, I remember this. Before, it was automatically enabling/disabling Raven as a whole if `NODE_ENV` wasn't production.

But yeah, you might be right. I don't think this is really used to mean an environment. I don't think someoen would say `NODE_ENV=staging`, etc.